Gemfile

```diff
@@ -11,12 +11,13 @@ gem 'jquery-rails'
 gem 'pg'
 gem 'bootstrap_form'
 gem 'bcrypt'
-
+gem 'fabrication'
+gem 'faker'
 
 
 group :development do
   gem 'thin'
-  gem "better_errors"
+  #gem "better_errors"
```

Comment:
No issues.  I commented out the "better_errors" gem when I tried to look at the log and find what where the parts of the params, but I couldn't find it and I thought this might be causing issues.  I should have uncommented it again although I'm not sure if i like the better-error error format